### DISCUSSION TITLE
How to display child issues of an issue from epics list ?

### DISCUSSION THREAD

**Discussion by davidjulien** [ankitpokhrel/jira-cli#232] [Q&A] [answered]
Hi,

Thank you for this cli! I'm discovering all its features and it looks great.

I have one question :

When you display all epics with 
```
jira epic list --order-by rank --reverse
```

and you switch to the right pane and you select one issue, how to display child issues related to this issue ?

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @davidjulien you can use `parent` filter to get child issues. However, it needs to be a new command and is not possible from the TUI at the moment.

```sh
# Get sub-tasks of a task
jira issue list --parent KEY-123
```

If you want to get list of issues in a particular epic, you can do:

```sh
# Get items in an epic
jira epic list EPIC-123
```

**Reply by davidjulien** [reply]
Thank you for your answer.
